docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -10,90 +10,82 @@ keywords: changelog, table, materialize
 
 ## Context
 
-You have a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog stream into a table. This is broadly called *materializing* a changelog stream into a table.
+You have a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog into a table. This is broadly called *materializing* a changelog into a table.
 
 ## In action
 
 ```sql
 CREATE TABLE t1 AS
-    SELECT k,
-           LATEST_BY_OFFSET(v1) AS v1,
-           LATEST_BY_OFFSET(v2) AS v2,
-           LATEST_BY_OFFSET(v3) AS v3
-    FROM s1
-    GROUP BY k
+    SELECT *
+    FROM changelog
     EMIT CHANGES;
 ```
 
-## Materializing a changelog stream
+## Materializing a changelog table
 
-In ksqlDB, you derive new tables by aggregating other streams and tables. To create a table that reflects the latest values for each key, use the `LATEST_BY_OFFSET` aggregation.
+In ksqlDB, you derive new tables by aggregating other streams and tables. To create a table that reflects the latest values for each key, declare the `changelog` as a table.
 
 Begin by telling ksqlDB to start all queries from the earliest point in each topic.
 
 ```sql
 SET 'auto.offset.reset' = 'earliest';
 ```
 
-Make a stream `s1` with four columns. `k` represents the key of the table. Rows with the same key represent information about the same entity. `v1`, `v2`, and `v3` are various value columns.
+Make a table `changelog` with four columns. `k` represents the key of the table. Rows with the same key represent information about the same entity. `v1`, `v2`, and `v3` are various value columns.
 
 ```sql
-CREATE STREAM s1 (
-    k VARCHAR KEY,
+CREATE TABLE changelog (
+    k VARCHAR PRIMARY KEY,
     v1 INT,
     v2 VARCHAR,
     v3 BOOLEAN
 ) WITH (
-    kafka_topic = 's1',
+    kafka_topic = 'changelog',
     partitions = 1,
-    value_format = 'avro'
+    value_format = 'JSON'
```

Comment:
chnaged to json to avoid `Cannot create topic 'changelog' with format AVRO without configuring 'ksql.schema.registry.url'` and remove the need to set up sr